src/components/TextInputFocusable/index.js

```diff
@@ -38,13 +39,13 @@ const propTypes = {
 
 const defaultProps = {
     maxLines: -1,
-    onPasteFile: () => {},
+    onPasteFile: () => { },
     shouldClear: false,
-    onClear: () => {},
+    onClear: () => { },
     style: null,
-    onDragEnter: () => {},
-    onDragLeave: () => {},
-    onDrop: () => {},
+    onDragEnter: () => { },
+    onDragLeave: () => { },
+    onDrop: () => { },
```

Comment:
Any reason you added the space here? If not let's revert them back to `{}` to be consistent with other files.

Reply:
The linter wanted to do it so I let it. I assumed it knew what it was doing. I'll revert those changes and re-submit.